scripts/docs_build

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Is the use/purpose of these three scripts noted somewhere?

Reply:
not really, and I hope they're temporary. all these scripts are a bit mysterious I expect :)